tests/test_daemon.cpp

```diff
@@ -1217,6 +1224,42 @@ TEST_F(Daemon, skips_over_instance_ghosts_in_db) // which will have been sometim
     mp::Daemon daemon{config_builder.build()};
 }
 
+TEST_P(ListIP, lists_with_ip)
+{
+    [[maybe_unused]] auto mock_factory = use_a_mock_vm_factory();
```

Comment:
Why `[[maybe_unused]]`? It's used in line 1235.

Reply:
Right, it was unused in an old version of the code. Removed.